### PR TITLE
Few minor clean ups

### PR DESCRIPTION
**Pull request by RyanAtViceSoftware**

### CODE REVIEW

File: docker-compose-debug.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ services:
       - ./db/init:/docker-entrypoint-initdb.d
     ports:
       - 5432
+    tty: true
```
Comment (RyanAtViceSoftware) [inline docker-compose-debug.yml:10]: Allows seeing colors in the output of docker-compose: https://github.com/docker/compose/issues/2231#issuecomment-165137408

File: docker-compose-ut.yml [hunk 2]
```diff
@@ -11,14 +11,26 @@ services:
...
+    healthcheck:
+      test: nc -z localhost 5432 || exit -1
+      interval: 5s
+      timeout: 5s
+      retries: 3
+      start_period: 5s
 
   api:
     image: acme/api:1.0
+    build: api
```
Comment (RyanAtViceSoftware) [inline docker-compose-ut.yml:24]: Couldn't get tests to run without this. @gnschenker I probably should have reread your blog post but just incase this is interesting.

File: docker-compose-debug.yml (hunk 1 above)
Comment (gnschenker) [reply docker-compose-debug.yml:10]: that is cool...

File: docker-compose-ut.yml (hunk 2 above)
Comment (gnschenker) [reply docker-compose-ut.yml:24]: yeah, the DB must be ready *before* the tests run :-)

**Review by gnschenker — COMMENTED** [review]
LGTM

**Pull request merged**